be/src/util/string_util.cpp

```diff
@@ -18,14 +18,15 @@
 #include "util/string_util.h"
 
 #include "gutil/strings/split.h"
+#include "util/hash_util.hpp"
 
 namespace doris {
 
 size_t hash_of_path(const std::string& identifier, const std::string& path) {
     size_t hash = std::hash<std::string>()(identifier);
     std::vector<std::string> path_parts = strings::Split(path, "/", strings::SkipWhitespace());
     for (auto& part : path_parts) {
-        boost::hash_combine<std::string>(hash, part);
+        HashUtil::hash_combine<std::string>(hash, part);
```

Comment:
Is it exactly same as `boost::hash_combine`?

Reply:
it is the same with boost hash_combine